build_projects/dotnet-host-build/build.sh

```diff
@@ -40,6 +42,9 @@ while [[ $# > 0 ]]; do
             # Allow CI to disable prereqs check since the CI has the pre-reqs but not ldconfig it seems
             export DOTNET_INSTALL_SKIP_PREREQS=1
             ;;
+        --hostbuildonly)
+            __HostBuildOnly=1
```

Comment:
I would suggest we rename this to "BuildDriver" since that is what dotnet-host-build is. Also, renaming it will alleviate any confusion with the actual host that is also built in this repo.